sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -366,12 +427,16 @@ async def create_database_if_not_exists(  # pylint: disable=redefined-builtin
             kwargs["initial_headers"] = initial_headers
         try:
             database_proxy = self.get_database_client(id)
-            await database_proxy.read(**kwargs)
-            return database_proxy
+            headers = await database_proxy.read(**kwargs)
```

Comment:
Nit: Just for readability as we use result in the other functions. There is also more than just the headers there. 
```suggestion
           result = await database_proxy.read(**kwargs)
```